certora/specs/ERC20.spec

```diff
@@ -110,6 +96,9 @@ rule transfer(env e) {
     uint256 recipientVotingPowerBefore = delegatedVotingPower(delegatee(recipient));
     uint256 otherBalanceBefore     = balanceOf(other);
 
+    // Safe require as it's proven by rule twoBalancesCannotExceedTotalSupply
+    require balanceOf(holder)+balanceOf(recipient) <= totalSupply();
```

Comment:
Same, this is only true if `holder != recipient`

Reply:
Fixed in https://github.com/morpho-org/morpho-token-upgradeable/pull/102/commits/faacfc3e99dfaabf152f5ecbf3a7940b392dcaaa.